src/ResourceManagement/Scheduler/SchedulerManagement/Microsoft.Azure.Management.Scheduler.nuspec

```diff
@@ -30,8 +30,8 @@
   </metadata>
   <files>
     <file src="ResourceManagement\Scheduler\SchedulerManagement\**\*.cs" target="src" />
-    <file src="..\binaries\net45\Microsoft.Azure.Management.Scheduler.dll" target="lib\dotnet" />
```

Comment:
Please set the target to `target="lib\dotnet"` and src to `..\binaries\portable\xxx`